contracts/TrustedForwarder.sol

```diff
@@ -28,15 +28,14 @@ contract TrustedForwarder is ITrustedForwarder {
     function verifyAndCall(GSNTypes.RelayRequest memory req, bytes memory sig)
     public
     override
-    returns (
-        bool success, bytes memory ret
-    ) {
+    {
         _verify(req, sig);
         _updateNonce(req);
 
         // solhint-disable-next-line avoid-low-level-calls
-        return req.target.call{gas:req.gasData.gasLimit}
-        (abi.encodePacked(req.encodedFunction, req.relayData.senderAddress));
+        (bool success, bytes memory returnValue) = req.target.call{gas:req.gasData.gasLimit}(abi.encodePacked(req.encodedFunction, req.relayData.senderAddress));
+        // TODO: use assembly to prevent double-wrapping of the revert reason (part of GSN-37)
+        require(success, string(returnValue));
```

Comment:
`GsnUtils.getError(returnValue)`

Reply:
Should I?

Reply:
yes, you should. assembly is optimization. using getError to actually get the error is bugfix.

Reply:
Fixed.